README.md

```diff
@@ -1,5 +1,5 @@
-# audits
-Public audits repository
+# Vacuumlabs smart contract audits
+Repository of smart contract audits performed by Vacuumlabs.
```

Comment:
```suggestion
# Vacuumlabs smart contract audit reports
Repository of smart contract audit reports performed by Vacuumlabs.
```

Reply:
This change would also need a change of the next heading: **List of Audits** -> **List of Audit Reports**